src/operator/controllers/intents_reconcilers/network_policy.go

```diff
@@ -117,6 +118,12 @@ func (r *NetworkPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		}
 	}
 
+	err = r.removeOrphanNetworkPolicies(ctx)
+	if err != nil {
+		r.RecordWarningEventf(intents, ReasonRemovingNetworkPolicyFailed, "could not remove orphan network policies: %s", err.Error())
```

Comment:
```suggestion
		r.RecordWarningEventf(intents, ReasonRemovingNetworkPolicyFailed, "failed to remove network policies: %s", err.Error())
```
Suggest omitting 'orphan' since it's not meaningful to the user (it's an implementation detail). This will remove network policies that are no longer referred to by the current set of intents